superset/base/kustomization.yaml

```diff
@@ -23,6 +26,11 @@ configMapGenerator:
     envs:
       - params.env
 
+secretGenerator:
```

Comment:
does it have to be a secret? seems like everything confidential is pulled in via env vars, so could be a config map no?

Reply:
I think that's a great idea. It's something I was thinking about too. Our deployment doesn't require it being a secret. If the op-first one doesn't either then I think a configmap would be good. I'll work on changing it.

Reply:
Done.